.github/workflows/run-e2e-tests.yml

```diff
@@ -0,0 +1,145 @@
+name: Run Terralab CLI e2e tests with BEE using GCP workspace
+on:
+  push:
+    branches:
+      - main
```

Comment:
so i know we went back and forth on this.  a cron job for this might be nice because if we change the service code this will let us know if we may have unknowingly broken the latest client.  Happy to chat about this.

We chatted with faces and decided to have a weekly run + merges to main

Reply:
good call - added schedule!